docs/source/contributing/documentation.rst

```diff
@@ -318,6 +318,23 @@ elements that are either a ``str`` or ``None``;
 ``(str, int)``; ``Tuple[:class:`int`, ...]`` for a tuple of variable
 length with only integers.
 
+.. note::
+   As a helper for tool for typesets, you can use ``typestring-parser`` which can 
+   be accessed by first downloading it via ``pip`` - ``pip install typestring-parser`` and
+   then using ``from typestring_parser import parse``.
+
+.. doctest::
+
+    >>>from typestring_parser import parse
```

Comment:
this line does not render correctly
![image](https://user-images.githubusercontent.com/49693820/116810321-c8dbf300-ab60-11eb-9e12-a61e630ee0ef.png)

Reply:
If I leave space for it, the CI failed (see https://github.com/ManimCommunity/manim/pull/1402/commits/d6036a0046ff5bdcc9769822f9a8826352495e18)
So I don't know what to do..

Reply:
You should disable doc test for that block of code.

Reply:
But then it will fail as this package is not in the dependency right?

Reply:
![image](https://user-images.githubusercontent.com/34582615/116813784-29742b80-ab73-11eb-953c-8652dffffade.png)

Reply:
That's what I wanted you to do :).